.analysis_options

```diff
@@ -1,8 +1,4 @@
 # Specify analysis options.
```

Comment:
cc @pq for the analysis_options changes.
My intent here is to make .analysis_options identical to flutter_analysis_options but without the comments, basically